app/components/find/courses/a_level_component/view.rb

```diff
@@ -32,6 +34,10 @@ def a_level_equivalency_summary_content
             I18n.t('find.courses.a_level.not_consider_a_level_equivalency')
           end
         end
+
+        def any_a_levels?
+          A_LEVEL_ATTRIBUTES.any? { |attribute| course.public_send(attribute).present? }
```

Comment:
Because we have this already in two place, I think we might need to move this to a method on course model? Or to the decorator? Wdyt?

Reply:
Agree, I'll find a place to put this